modules/appgw/main.tf

```diff
@@ -40,6 +40,7 @@ resource "azurerm_public_ip" "this" {
 
   sku               = "Standard"
   allocation_method = "Static"
+  domain_name_label = try(var.domain_name_label, null)
```

Comment:
If we have default value for ``domain_name_label``, then can we skip ``try`` function ?